project/Dependencies.scala

```diff
@@ -83,6 +83,8 @@ object Dependencies {
     // Added explicitly for when artery tcp is used
     val agrona = "org.agrona" % "agrona" % agronaVersion // ApacheV2
 
+    val asnOne = "com.hierynomus" % "asn-one" % "0.4.0"
```

Comment:
add the `// ApacheV2` comment

(https://gitlab.com/hierynomus/asn-one/-/blob/master/LICENSE)